src/io/fabric8/Fabric8Commands.groovy

```diff
@@ -496,8 +496,12 @@ def getServiceURL(String serviceName, String namespace = null, String protocol =
 }
 
 def isOpenShiftS2I() {
-    def openshiftYaml = "target/classes/META-INF/fabric8/openshift.yml"
     try {
+        new File('.').eachFileRecurse(FILES) {
```

Comment:
Yes my bad, I missed the import statement `import static groovy.io.FileType.FILES`. I kinda did a mock testing where I had imported it.